src/components/connect-create-buttons.tsx

```diff
@@ -1,71 +1,84 @@
 import { __ } from '@wordpress/i18n';
+import { useOffline } from '../hooks/use-offline';
 import { cx } from '../lib/cx';
 import { getIpcApi } from '../lib/get-ipc-api';
 import { ArrowIcon } from './arrow-icon';
 import Button, { ButtonVariant } from './button';
 import offlineIcon from './offline-icon';
 import { Tooltip } from './tooltip';
 
-interface ConnectCreateButtonsProps {
-	connectSite: () => void;
-	isOffline: boolean;
+interface ConnectButtonProps {
 	connectButtonVariant: ButtonVariant;
-	createButtonVariant: ButtonVariant;
+	connectSite?: () => void;
 	disableConnectButtonStyle?: boolean;
+	className?: string;
+}
+
+interface CreateButtonProps {
+	createButtonVariant: ButtonVariant;
 	selectedSite: SiteDetails;
+	createButtonText?: string;
+	className?: string;
 }
 
-export const ConnectCreateButtons = ( {
-	connectSite,
-	isOffline,
-	createButtonVariant,
+export const ConnectButton = ( {
 	connectButtonVariant,
+	connectSite,
 	disableConnectButtonStyle,
-	selectedSite,
-}: ConnectCreateButtonsProps ) => {
+	className,
+}: ConnectButtonProps ) => {
+	const isOffline = useOffline();
```

Comment:
I appreciate the move of `isOffline` inside the components so we don't have to pass it as a prop. I think it makes the code more readable.

Reply:
Yeah I realized that we are passing isOffline as prop for no apparent reason 😅